### PR TITLE
Fix kpm commands install folder

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/aspnet/XRE/issues/1268

@troydai @ChengTian can you please review

@tugberkugurlu here is the fix if you want to try it :)

### CODE REVIEW

**Comment by davidfowl** [comment]
What does this do?

**Comment by victorhurdugaci** [comment]
The bug: `kpm commands install` installs in  `%userprofile%\bin` rather than  `%userprofile%\.k\bin`.
This change: fixes that and installs in the right folder

File: src/Microsoft.Framework.PackageManager/Install/AppCommandsFolderRepository.cs [hunk 1]
```diff
@@ -140,7 +140,16 @@ public static AppCommandsFolderRepository CreateDefault()
             {
                 userProfileFolder = Environment.GetEnvironmentVariable("HOME");
             }
-            var binFolder = Path.Combine(userProfileFolder, "bin");
+
```
Comment (ChengTian) [inline src/Microsoft.Framework.PackageManager/Install/AppCommandsFolderRepository.cs]: Add TODO comments?
```c#
// TODO: use Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) when it's available on CoreCLR
```

Comment (victorhurdugaci) [inline src/Microsoft.Framework.PackageManager/Install/AppCommandsFolderRepository.cs]: Done

**Comment by ChengTian** [comment]
:shipit:

**Pull request merged**